src/components/Layout/Layout.tsx

```diff
@@ -23,7 +23,7 @@ export function DashboardLayout({ children }) {
           </div>
         </aside>
         {/* Main content */}
-        <div className="h-screen overflow-y-auto flex-col mb-3">
+        <div className="h-screen overflow-y-auto flex-col">
```

Comment:
flex-col may be used with flex, otherwise it has no effect

Reply:
```suggestion
        <div className="h-screen overflow-y-auto">
```

You're right, that wasn't the aim of this PR but we can remove it too.